service/rest_task.go

```diff
@@ -36,6 +36,7 @@ type RestTask struct {
 	Activated           bool                  `json:"activated"`
 	BuildId             string                `json:"build_id"`
 	DistroId            string                `json:"distro"`
+	Container           string                `json:"container"`
```

Comment:
Should this be omitempty for the JSON tag? It won't be set for regular host tasks.